lib/locreator.js

```diff
@@ -12,6 +13,15 @@ var Locreator = function (nemo) {
   this.drivex = Drivex(nemo.driver, nemo.wd);
   this.locatex = Locatex(nemo);
   this.normify = this.normalize = function (locator) {
+    if(locator.constructor !== String) {
+      if(!locator.locator || !locator.locator.trim()) {
+        throw new Error('NO or EMPTY locator found, please fix it');
+      }
+      if(!locator.type || !locator.type.trim() || !nemo.wd.By[locator.type]) {
```

Comment:
I wanted to be consistent with our existing approach mentioned in locator definition [here](https://github.com/paypal/nemo-view#locatordefinition). Where we say
`String of the form <type>:<locator>or <locator> (where <type> will be assumed as css)`